routers/web/repo/issue.go

```diff
@@ -1865,6 +1912,34 @@ func UpdateIssueAssignee(ctx *context.Context) {
 	})
 }
 
+// UpdateIssueConfidential change issue's confidential
+func UpdateIssueConfidential(ctx *context.Context) {
+	issue := GetActionIssue(ctx)
+	if ctx.Written() {
+		return
+	}
+
+	if !ctx.IsSigned || (!issue.IsPoster(ctx.User.ID) && !ctx.Repo.CanWriteIssuesOrPulls(issue.IsPull)) {
+		ctx.Error(http.StatusForbidden)
+		return
+	}
+
+	title := ctx.FormTrim("title")
+	if len(title) == 0 {
+		ctx.Error(http.StatusNoContent)
+		return
+	}
+
+	if err := issue_service.ChangeTitle(issue, ctx.User, title); err != nil {
+		ctx.ServerError("ChangeTitle", err)
+		return
+	}
```

Comment:
WIP?

Reply:
Because I think you intended to change the issue confidentiality status here, not the title.

Reply:
Left-over from trying to write the function `IssuePrivate`